frontend/src/components/calendar/event-dialog/EventDialog.js

```diff
@@ -0,0 +1,109 @@
+import './EventDialog.css';
+
+import {
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  Button,
+  Tooltip
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import CloseIcon from '@material-ui/icons/Close';
+import { getAMPMTimeRange, getDayMonthDateStr } from '../../../utility/date-time';
+import { storeContext } from '../../../store/Store';
+import { useContext, useState } from 'react';
+import React from 'react';
+import CreateEvent from './create-event/CreateEvent';
+
+const useStyles = makeStyles({
+  paper: {
+    width: '40%',
+  },
+});
+
+function VolunteerInfo(props) {
+  const { volunteer } = props;
+
+  return (
+    <li>
+      {volunteer.name}: <span className="volunteer-time">{getAMPMTimeRange(volunteer.start, volunteer.end)}</span>
+    </li>
+  );
+}
+
+function EventDialog(props) {
+  const { event } = props;
+  const classes = useStyles();
+  const [{ user }, ] = useContext(storeContext);
```

Comment:
[`useSelector`](https://react-redux.js.org/api/hooks#useselector) might be a better hook to use.

Reply:
Since we aren't using redux (yet!), this can be a future task.

Reply:
Sounds good